### PR TITLE
Update base layout and remove unnecessary code

### LINKED ISSUE #4: layout base.Portfile
A few suggestions for the ```base.Portfile``` template:

1. adding a comment along the lines of "Portfile generated/maintained using the Universal Packaging Tool (upt)" would be useful to indicate which ports are using ```upt```. (I would add it just before the ```name``` field).
2. rephrase the comment about ```supported_archs``` for more clarity. How about something like "# uncomment the following line if no architecture-dependent files are installed, otherwise remove"
3. remove lines 16 (```print(self._depends)```) and 25 (```# env.filters['reqformat'] = self.jinja2_reqformat```) as they seem unnecessary 
4. add ```categories` field (probably commented out since it's difficult to fill in sensible values automatically)
5. add "default" example for "maintainers"; ```maintainers: {example.org:julesverne @GitHub-handle} openmaintainer```
6. follow the order of fields from [the guide](https://guide.macports.org/#development.creating-portfile) a bit more closely. Admittedly, some of this is personal preference so it doesn't _need_ to be changed, but please have a look. I would recommend the following order:

```
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4

PortSystem          
PortGroup           

name                
version             
revision            

# categories-append
platforms           
# uncomment the following line if no architecture-dependent files are installed, otherwise remove
# supported_archs     noarch
license            
maintainers: {example.org:julesverne @GitHub-handle} openmaintainer        

description         
long_description    

homepage            
master_sites        
distname            
# if needed, add keyword depending on compression type of archive
checksums           
```

### PR DESCRIPTION
**Pull request by Korusuke**
Fixes #4

### CODE REVIEW

**Comment by Steap** [comment]
It is kind of weird to have a commit titled "Merge branch 'master' into base_port". What happened?

You could add "Fixes #4" at the end of the commit log. I think it should close the corresponding issue automagically once we merge the PR.

**Comment by reneeotten** [comment]
@Korusuke we don't want to have "Merge" commits in the git history.

You probably did ```git pull```; instead you should use ```git pull --rebase``` which will add your new commits on top of the ```master``` branch. Please see the documentation: https://trac.macports.org/wiki/WorkingWithGit#updating. 

Try to recover from this first (a good way of working on your git-vodoo a bit more) and then we can review the actual content. Do make a backup first (```git checkout base_port ; git checkout -b backup```, go back to your branch ```git checkout base_port``` and try to fix it).

**Comment by Korusuke** [comment]
A merge conflict was showing up due to the newly merged #12 so I just tried solving it using the git GUI which now I guess I shouldn't have done, will fix it in few minutes.

**Comment by reneeotten** [comment]
> A merge conflict was showing up due to the newly merged #12 so I just tried solving it using the git GUI which now I guess I shouldn't have done, will fix it in few minutes.

something is not right yet as you have duplicate fields in the template now....

File: upt_macports/templates/base.Portfile
Comment (Steap) [inline upt_macports/templates/base.Portfile]: You could add "XXX" or "TODO" at the end of this line, so that it is clearly visible to the maintainer when they open the file. @reneeotten what do you think?

Comment (reneeotten) [reply upt_macports/templates/base.Portfile]: I agree - we should signal in a clear way that a maintainer is supposed to do something about it. I think we should add one link to the MacPorts guide on top of that block in the Portfile, comment out lines where action is required, and add TODO at the end of the line. I have made some suggestions in the the base template.

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [inline upt_macports/templates/base.Portfile]: let's not add an new line here

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [inline upt_macports/templates/base.Portfile]: add one line here to indicate where a maintainer can bind more information: 
```
# Please adapt the lines with a "TODO" statement and check the guide for more information:
# https://guide.macports.org/#reference.keywords 
```

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [inline upt_macports/templates/base.Portfile]: add "TODO", like: ```# TODO: uncomment the .....```

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [inline upt_macports/templates/base.Portfile]: comment this line out and add TODO at the end:
```# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer # TODO```

**Review by reneeotten — CHANGES_REQUESTED** [review]
a suggestions to follow-up on what @Steap already mentioned, besides that it looks good to go!

**Comment by Korusuke** [comment]
I have done the necessary changes and will update the Portfile over at macports/ports once this PR is merged.

**Comment by reneeotten** [comment]
this looks good to me and it can always be fine-tuned based on input from the MacPorts community later. @Steap, since it's really Portfile/MacPorts related and your suggestions incorporated I'm going to assume that your fine with me merging it ;)

File: upt_macports/templates/base.Portfile
Comment (mojca) [inline upt_macports/templates/base.Portfile:7]: I would probably use `maintained` only after we get automatic updates working :)

File: upt_macports/templates/base.Portfile
Comment (mojca) [inline upt_macports/templates/base.Portfile:19]: I would also keep a line for `nomaintainer` and let the user decide between the two.
It doesn't serve anyone to force users to assume maintainership when they are not comfortable to do so and not willing to take care of the port later.

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [reply upt_macports/templates/base.Portfile:7]: we are optimistic about it ;)

File: upt_macports/templates/base.Portfile
Comment (reneeotten) [reply upt_macports/templates/base.Portfile:19]: yes, that's a valid point - how about something like this?
```
# TODO: either add your e-mail address and GitHub handle or set to nomaintainer
# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer
# maintainers         nomaintainer
```

Comment (mojca) [reply upt_macports/templates/base.Portfile:19]: One further idea. At the end the port would ideally not contain any comments. So maybe the `TODO`s are a bit superfluous. Maybe something along those lines?

```
## If you would like to keep maintaining this port in the future as described in
##     https://guide.macports.org/chunked/project.contributing.html
## please add your contacts (e-mail, github handle), else keep `nomaintainer`.
# maintainers         {example.org:julesverne @github-handle} openmaintainer
# maintainers         nomaintainer
```

And then also ask users in a comment somewhere on top to go through the commented-out sections and check them out (and ideally remove most comments).

**Pull request merged**
